src/test/java/com/hyperlocal/server/MerchantControllerTest.java

```diff
@@ -8,6 +8,7 @@
 import static org.mockito.Mockito.when;
```

Comment:
You didn't have to make changes to ShopControllerTest class?

Reply:
Short: No but now that I think of it, I should.
Long:  My understanding is, upsert merchant and upsert shop methods were both using helper functions to make calls to database. The unit tests were written specifically for these helper functions. In this PR, I changed unit tests for upsert merchant methods to test the actual methods being called and removed helper functions altogether; the helpers were small and not reused anywhere. I did not remove helper functions from upsert shop methods. On it right now.

Reply:
This file https://github.com/googleinterns/hyperlocal-gpay-spot/blob/master/src/test/java/com/hyperlocal/server/ShopControllerTest.java